src/MoBi.Core/OSPSuite.Core/Domain/Builder/PathAndValueEntity.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.Linq;
+using OSPSuite.Core.Domain.Formulas;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.Core.Domain.UnitSystem;
+
+namespace OSPSuite.Core.Domain.Builder
```

Comment:
I think this is what you had in mind with a common object. Basically, we'd just need to rename the x.StartValue to x.Value all over the place in Core and MoBi.

Reply:
Renaming probably not at the MoleculeSV and PSV level as we'll have to write converter
Or we can do it but we need to start adding an ongoing task for stuff to convert
Alternatively you have Value in the base class and keep StartValue <=> Value